deployment/run-unit-tests.sh

```diff
@@ -47,15 +47,20 @@ source $venv_folder/bin/activate
 cd $source_dir
 pip install --upgrade pip
 pip install -r requirements-dev.txt
+pip install -r api/tests/requirements-test.txt
 
 # env variables
 export PYTHONDONTWRITEBYTECODE=1
-export AMC_ENDPOINT_URL="AmcEndpointUrl"
-export AMC_API_ROLE_ARN="AmcApiRoleArn"
+export AMC_ENDPOINT_URL="https://test_end_point_url.com/test"
+export AMC_API_ROLE_ARN="arn:aws:iam::999999999999:role/SomeTestRole"
 export SOLUTION_NAME="SOLUTION_NAME"
 export SOLUTION_VERSION="SOLUTION_VERSION"
+export VERSION="0.0.1"
```

Comment:
0.0.0 would a better dummy version because 0.0.1 could actually exist.

Reply:
Are SOLUTION_VERSION and VERSION redundant?

Reply:
you right i will take it out